go.mod

```diff
@@ -108,3 +107,5 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 	k8s.io/klog/v2 v2.130.1 // indirect
 )
+
+replace github.com/sigstore/sigstore => github.com/trail-of-forks/sigstore v0.0.0-20250305100229-6bb0c7c5e889
```

Comment:
This will be removed once that branch is merged into `sigstore/sigstore` right?

Reply:
Yes! I have split those in separate commits marked with `REMOVE ME` :)